conditional/params.go

```diff
@@ -0,0 +1,137 @@
+package conditional
+
+import (
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/istreamlabs/huma"
+)
+
+// trimETag removes the quotes and `W/` prefix for incoming ETag values to
+// make comparisons easier.
+func trimETag(value string) string {
+	if strings.HasPrefix(value, "W/") && len(value) > 2 {
+		value = value[2:]
+	}
+	return strings.Trim(value, "\"")
+}
+
+// Params allow clients to send ETags or times to make a read or
```

Comment:
```suggestion
// Params allow clients to send ETags or timestamps to make a read or
```
I get that these are `time.Time`s on the server, but my brain stumbled on this sentence--is it accurate to call them timestamps or datetimes?